src/modules/authentication.json

```diff
@@ -112,6 +125,134 @@
           }
         }
       ]
+    },
+    {
+      "name": "device",
+      "summary": "Get a device token scoped to the current app.",
+      "tags": [
+        {
+          "name": "capabilities",
+          "x-uses": [
+            "xrn:firebolt:capability:token:device"
+          ]
+        }
+      ],
+      "params": [
+      ],
+      "result": {
+        "name": "token",
+        "summary": "the token value and expiration",
+        "schema": {
+          "type": "string"
+        }
+      },
+      "examples": [
+        {
+          "name": "Acquire a Firebolt device identity (XACT) token",
```

Comment:
Wonder if we should not put XACT here, since that is a comcastic term

Reply:
good catch, fixed